store/models.py

```diff
@@ -30,3 +30,7 @@ def __str__(self):
         else:
             return f'{self.book.title} - Available'
 
+class BRating(models.Model):
+    critic = models.ForeignKey(User, related_name='critic', null=True, blank=True, on_delete=models.SET_NULL)
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    rating=models.FloatField(default=0.0)
```

Comment:
As per instructions this should be integer. But using a separate model was nice.